CSharp/BatchExplorer/Models/TaskModel.cs

```diff
@@ -51,6 +51,12 @@ public class TaskModel : ModelBase
         public string CommandLine { get { return this.Task.CommandLine; } }
 
         /// <summary>
+        /// The exit code of the task
+        /// </summary>
+        [ChangeTracked(ModelRefreshType.Basic)]
+        public int? ExitCode { get { return this.Task.ExecutionInformation.ExitCode; } }
```

Comment:
Fixed. This is safer even if I didn't see an issue even with Active (non-executed jobs) in testing